acrobat/scripts/scripts.js

```diff
@@ -340,3 +340,15 @@ const { ietf } = getLocale(locales);
     }
   }, 1000);
 }());
+
+const isCommerceActive = () => {
+  const metaTag = document.querySelector('meta[name="commerce"]');
```

Comment:
one more minor thing. MILO core has a util for getting the metadata info 
https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L169C17-L169C28